ff/serving/provider/online.go

```diff
@@ -221,3 +357,75 @@ func (table redisOnlineTable) Get(entity string) (interface{}, error) {
 	}
 	return result, nil
 }
+
+// CASSANDRA SET ENTITY
+func (table cassandraOnlineTable) Set(entity string, value interface{}) error {
+
+	table.session.SetConsistency(gocql.One)
+
+	//Get table key
+	key := table.key
+	var keyName string
+	keyName = sn.Custom(key.String(), "[^a-zA-Z0-9_]")
+
+	//Get table Name
+	var tableName string
+	tableName = fmt.Sprintf("%s.table%s", key.Keyspace, sn.Custom(key.Feature, "[^a-zA-Z0-9_]"))
+
+	//Set entity
+	val := fmt.Sprintf("%v", value)
```

Comment:
Not sure if this will work on all value types

Reply:
Is this necessary or can value be set directly?